var/spack/repos/builtin/packages/py-cymem/package.py

```diff
@@ -13,7 +13,9 @@ class PyCymem(PythonPackage):
     homepage = "https://github.com/explosion/cymem"
     pypi = "cymem/cymem-2.0.3.tar.gz"
 
+    version("2.0.7", sha256="e6034badb5dd4e10344211c81f16505a55553a7164adc314c75bd80cf07e57a8")
     version("2.0.3", sha256="5083b2ab5fe13ced094a82e0df465e2dbbd9b1c013288888035e24fd6eb4ed01")
 
     depends_on("py-setuptools", type="build")
     depends_on("py-wheel@0.32.0:0.32", type="build")
+    depends_on("py-cython@0.25:", type="build", when="@2.0.7:")
```

Comment:
```suggestion
    depends_on("py-cython@0.25:", when="@2.0.7:",  type="build")
```
to keep it consistent with most of the other python packages

Reply:
This is optional. It used to be that the former was much more popular. I started doing the latter to match the parameter order in the `depends_on` definition and others have followed suit.